aea/configurations/validation.py

```diff
@@ -243,7 +243,7 @@ def _validate(self, instance: Dict) -> None:
     @staticmethod
     def _build_message_from_errors(errors: List[jsonschema.ValidationError]) -> str:
         """Build an error message from validation errors."""
-        path = lambda error: ".".join(list(error.path))  # noqa # type: ignore
+        path = lambda error: "::".join(map(str, error.path))  # noqa
```

Comment:
error was on constructing error message string.

Reply:
during the investigation another place of improvement was found:
https://github.com/valory-xyz/open-aea/issues/576